app/src/main/java/com/example/easysurvey/service/SurveyService.kt

```diff
@@ -0,0 +1,21 @@
+package com.example.easysurvey.service
+
+import com.example.easysurvey.RetrofitClientInstance
+import com.example.easysurvey.dao.ISurveyDao
+import com.example.easysurvey.dto.Survey
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.async
+import kotlinx.coroutines.withContext
+import retrofit2.awaitResponse
+
+class SurveyService {
+
+    suspend fun fetchSurveys() : List<Survey>? {
```

Comment:
Created a fetchSurveys function.  You can use this function to test whether or not the JSON file you implement has actually worked

Reply:
This looks like an incomplete feature.
Are you using these anywhere?  I don't see them used.
If not in use, remove them.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.